pkg/security/ebpf/c/commit_creds.h

```diff
@@ -0,0 +1,272 @@
+#ifndef _COMMIT_CREDS_H_
+#define _COMMIT_CREDS_H_
+
+struct credentials_event_t {
```

Comment:
It may be clearer to have 3 separate events

Reply:
I added a union to make it clearer without having to create different events 😄